client/src/components/screen/ProfileUpdate.tsx

```diff
@@ -64,6 +66,13 @@ const ProfileImage = styled.Image`
   border-radius: 45px;
 `;
 
+declare let Blob: {
+  prototype: Blob;
+  new (): Blob;
+  new (blobParts: Array<string>, options: Record<string, string>): Blob;
+  name: string;
+};
+
```

Comment:
Could you kindly refer to any resource on having this approach? I think It is good but I want to read more about this!

Reply:
Sure, first I followed the Blob example to upload a file from the client (See: https://github.com/jaydenseric/apollo-upload-client/#Blob). Then I need to define the Blob type, I get some hints from this to references.
- [Interface File](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.file.html)
- [Blob in TypeScript](https://stackoverflow.com/questions/13124080/blob-in-typescript)